network-area-diagram-viewer/pom.xml

```
@@ -42,8 +42,8 @@
         <junit.version>4.13.1</junit.version>
         <maven.compiler.source>17</maven.compiler.source>
         <maven.compiler.target>17</maven.compiler.target>
-        <powsybl.diagram.version>3.1.0-SNAPSHOT</powsybl.diagram.version>
-        <powsybl.core.version>5.0.0</powsybl.core.version>
+        <powsybl.diagram.version>3.1.0</powsybl.diagram.version>
```

```suggestion
        <powsybl.diagram.version>3.2.0-SNAPSHOT</powsybl.diagram.version>
```

The primary goal of these viewers is to be used with latest snapshot version of tested artifact, so you shouldn't refer to a release.